test/test_torch.py

```diff
@@ -17135,7 +17139,7 @@ def test_lcm(self, device, dtype):
     @slowTest
     @onlyOnCPUAndCUDA
     @dtypes(torch.float32, torch.float64, torch.bfloat16, torch.int32, torch.int64, torch.cfloat, torch.cdouble)
-    @dtypesIfCUDA(torch.float32, torch.float64)
+    @dtypesIfCUDA(torch.float32, torch.float64, torch.cfloat, torch.cdouble)
     @tf32_on_and_off(0.01)
     def test_mm(self, device, dtype):
         def _test_mm(n, m, p, dtype, genf):
```

Comment:
LOL, GitHub UI is so confusing.

<img width="492" alt="Screenshot_20200731_131828" src="https://user-images.githubusercontent.com/1032377/89074055-7c912780-d330-11ea-9304-f9ef9502a4e3.png">